src/promptflow-devkit/promptflow/_sdk/_utilities/chat_utils.py

```diff
@@ -10,12 +17,72 @@ def construct_flow_absolute_path(flow: str) -> str:
 
 
 # Todo: use base64 encode for now, will consider whether need use encryption or use db to store flow path info
-def construct_chat_page_url(flow, port, url_params, enable_internal_features=False):
-    flow_path_dir, flow_path_file = resolve_flow_path(flow)
-    flow_path = str(flow_path_dir / flow_path_file)
```

Comment:
maybe we need keep this since ux need dag/flex yaml path, but cli can pass flow directory